spec/datadog/tracing/contrib/sidekiq/support/helper.rb

```diff
@@ -89,4 +89,22 @@ def expect_in_sidekiq_server(duration: 2, &expectations)
     app_tempfile.close
     app_tempfile.unlink
   end
+
+  def run_sidekiq_server
+    app_tempfile = Tempfile.new(['sidekiq-server-app', '.rb'])
+
+    require 'sidekiq/cli'
+
+    configure_sidekiq
+    # binding.pry
```

Comment:
Weird that rubocop didn't catch that 🤔

Reply:
Thanks for pointing this out 😓 it has been removed now!